packages/generators/generators/lib/files/ts/plugin/tsconfig.server.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "@strapi/typescript-utils/tsconfigs/server",
+
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "."
+  },
+
+  "include": [
+    // Include the root directory
+    "server",
+    // Force the JSON files in the src folder to be included
+    "server/**/*.json"
```

Comment:
As far as I remember these paths have created quite a few problems on windows machines. @Convly could you have a look?

Reply:
These paths aren't different from what they are now? and afaik TS handles these internally regardless of how you write them?

Reply:
I don't know anything about generators and the way they configure TS - just remembered there were problems at some point. 🤷🏼

Reply:
Internally TS automatically parse `/` and `\` 
https://github.com/microsoft/TypeScript/blob/main/src/compiler/path.ts
So _in theory_ we should not have an issue